supertokens/supertokens-mysql-plugin
# Docker stderr

## Summary of change
When running in a container without `$ERROR_LOG_PATH` specified, this change will now print errors to `stderr` (instead of being merged into `stdout`

## Related issues
- https://github.com/supertokens/supertokens-core/issues/5

## Test Plan
1. Run container with incorrect port: `sudo docker run --network="host" -p 3567:3567 -e MYSQL_USER="root" -e MYSQL_HOST="127.0.0.1" -e MYSQL_PORT="3307" -e MYSQL_PASSWORD="root" supertokens 2> >(while read line; do echo -e "\e[01;31m$line\e[0m" >&2; done)`
2. Confirm error in red

![image](https://user-images.githubusercontent.com/603829/150676078-ee97ee52-675b-4940-974e-f994a7cede02.png)

## Documentation changes
None

## Checklist for important updates
   - In `build.gradle`
   - To know which one it is, run find the latest released tag (`git tag`) in the format `vX.Y.Z`, and then find the latest branch (`git branch --all`) whose `X.Y` is greater than the latest released tag.
   - If no such branch exists, then create one from the latest released branch.

## Remaining TODOs for this PR
None
